### PR TITLE
Feature/tagging

### PR DESCRIPTION
**Pull request by NiklasCorneliusKunz**
Ihr könnt gerne mal das Taggen am Ende des Verifikationsprozesses ausprobieren und mir rückmelden, ob das so passt.

### CODE REVIEW

File: src/app/review-item/components/tags-question/tags-question.component.ts [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    if (index >= 0) {
+      this.tagsUser.splice(index, 1);
+    }
+  }
+
+  submitTags() {
+    this.loader.show();
+    this.itemsService.setItemTags(this.itemId, this.tagsUser);
+    this.loader.hide();
+    this.finished.emit();
```
Comment (SandroGlueck) [inline src/app/review-item/components/tags-question/tags-question.component.ts:72]: Shouldn't you wait for the Promise to be resolved before going on?

**Review by SandroGlueck — CHANGES_REQUESTED** [review]
Could you please add the package-lock.json in the .gitignore as it messes with yarn.

That way you can continue to use npm locally and yarn can be used in CI.

File: src/app/archive/archive.module.ts [hunk 2]
```diff
@@ -1,21 +1,24 @@
...
-    ]
+  imports: [
+    CommonModule,
+    MaterialModule,
+    RecaptchaModule,
+    RecaptchaFormsModule,
+    HelperModule,
+    RouterModule,
+    MatChipsModule,
+    MatInputModule,
```
[the remark this answers is not in the document]
Comment (NiklasCorneliusKunz) [reply src/app/archive/archive.module.ts:21]: Thanks for the info.

File: dk.code-workspace [hunk 3]
```diff
@@ -0,0 +1,7 @@
+{
+  "folders": [
+	{
+		"path": "."
+	}
+]
+}
```
[the remark this answers is not in the document]
Comment (NiklasCorneliusKunz) [reply dk.code-workspace:7]: Comes from VS Code. I will put it on gitignore.

File: src/app/review-item/components/tags-question/tags-question.component.ts (hunk 1 above)
Comment (NiklasCorneliusKunz) [reply src/app/review-item/components/tags-question/tags-question.component.ts:72]: Yes, of course.

**Review by SandroGlueck — APPROVED** [review]

**Pull request merged**